### PR TITLE
Add basic db models

### PR DESCRIPTION
**Pull request by finloop**

### CODE REVIEW

File: instance/config.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Config:
+    """
+    Configuration class
+
+    ...
```
Comment (pgronkievitz) [inline instance/config.py:5]: get rid of `...` and unnecessary blank lines (one is enough)

File: manage.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from flask_migrate import Migrate
```
Comment (pgronkievitz) [inline manage.py:1]: write file docs

File: instance/config.py (hunk 1 above)
Comment (pgronkievitz) [inline instance/config.py:5]: get rid of `...` and unnecessary blank lines (one is enough)

File: models.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
+from sqlalchemy.ext.hybrid import hybrid_property
```
Comment (pgronkievitz) [inline models.py:1]: write file docs

File: models.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+from app import BCRYPT
+from app import DB
+from app import LOGIN_MANAGER
+
+
+class User(DB.Model):
```
Comment (pgronkievitz) [inline models.py:8]: class docs

File: models.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        """
+        if isinstance(other, User):
+            return self.get_id() == other.get_id()
+        return False
+
+    def __ne__(self, other):
+        """
+        Checks the inequality of two `User` objects using `get_id`.
+        """
+        equal = self.__eq__(other)
```
Comment (pgronkievitz) [inline models.py:54]: make oneliner out of this

File: models.py [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        return not equal
+
+    def __hash__(self):
+        """
+        Returns hash of an object.
+        """
+        return hash((self.id, self.email))
+
+
+class Reservation(DB.Model):
```
Comment (pgronkievitz) [inline models.py:64]: class docs

File: models.py [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+    user_id = DB.Column(DB.Integer, DB.ForeignKey("user.id"), nullable=False)
+    server_id = DB.Column(DB.Integer,
+                          DB.ForeignKey("server.id"),
+                          nullable=False)
+
+    def __repr__(self):
+        return "<Reservation {} for {}>".format(self.id, self.user_id)
+
+
+class Server(DB.Model):
```
Comment (pgronkievitz) [inline models.py:80]: class docs

File: models.py [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+        return str(self.id)
+
+    def __eq__(self, other):
+        """Check the equality of two `User` objects using `get_id`."""
+        if isinstance(other, User):
+            return self.get_id() == other.get_id()
+        return False
+
+    def __ne__(self, other):
+        """
```
Comment (pgronkievitz) [inline models.py:81]: Lacks documentation what `other` is

File: manage.py (hunk 2 above)
Comment (finloop) [reply manage.py:28]: Done.

File: models.py (hunk 8 above)
Comment (finloop) [reply models.py:112]: Added.

File: models.py (hunk 5 above)
Comment (finloop) [reply models.py:120]: Added docs for `other` param, so it cannot be turned into oneliner.

File: models.py (hunk 7 above)
Comment (finloop) [reply models.py:168]: Done

File: instance/config.py (hunk 1 above)
Comment (finloop) [reply instance/config.py]: done

File: models.py (hunk 4 above)
Comment (finloop) [reply models.py:8]: done

**Pull request merged**